ebu_tt_live/ui/user_input_producer/uip.js

```diff
@@ -479,23 +488,25 @@ $(document).ready(function() {
             var scheduled_time = new Date(Date.now());
             scheduled_time_input = $("#scheduled-time-input").val();
             var scheduled_time_parsed = scheduled_time_input.match(/(\d\d):(\d\d):(\d\d)/);
-            if (scheduled_time_parsed == null)
-            return 0;
-            scheduled_time.setHours(scheduled_time_parsed[1]);
-            scheduled_time.setMinutes(scheduled_time_parsed[2]);
-            scheduled_time.setSeconds(scheduled_time_parsed[3]);
-            timeout = scheduled_time.getTime() - Date.now();
+            if (scheduled_time_parsed == null) {
+              return 0;
+              scheduled_time.setHours(scheduled_time_parsed[1]);
```

Comment:
Why would there be more lines after the return? I don't think these were supposed to be handled as part of the `if`...